automl/AutoML.sln

```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Folder structure would usually be automl/api/AutoML.slm (see KMS and follow that).
In general follow folder structure and project naming as in the KMS samples.